File: 05.fundamentalDatatypes/exercise1.pl
use strict;
use diagnostics;
use warnings;

print("Hello, Enter your age in years?\n");
my $age = <>;
chomp($age);
my $days = $age * 365;
chomp($days);
my $hours = $days * 24;
chomp($hours);
my $minutes = $hours * 60;
chomp($minutes);
my $seconds = $minutes * 60;
chomp($seconds);
print("So you are $age years old, or\n$days days old, or\n$hours hours old, or\n$minutes minutes old, or\n$seconds seconds old.\n")

File: 05.fundamentalDatatypes/Exercise_1.cpp
#include <iostream>

using namespace std;

int main(){

	int age = 0;
	
	cout<< "Eneter your age in years: ";
	cin>>age;

	int days = age * 365;
	int hours = age * 8760;
	int minutes = age * 525600;
	int seconds = age * 31540000;

	cout<< "You are ... \n" << days << " days old, or \n" << hours << " hours old, or \n" << minutes << " minutes old, or \n" << seconds << " seconds old.\n";
		
	return 0;
	
}
